crates/ruff_python_codegen/src/generator.rs

```diff
@@ -1795,47 +1770,37 @@ if True:
     #[test]
     fn set_quote() {
```

Comment:
maybe rename the test function?

```suggestion
    fn quotes_are_preserved() {
```

Reply:
I also thought about moving these to the existing `quote` test. Would that be better, or is it worth keeping them separate?

`quote` uses `round_trip` instead of `round_trip_with`, but we're not really utilizing `round_trip_with` now either.

Reply:
Yeah, I think we can merge the tests now (or delete this function entirely if it is now completely redundant). As we discussed previously, the main purpose of this function was to test that setting the `quote` field worked correctly. But the `quote` field has now gone! :-D